src/app/content/utils/contentManipulation.ts

```diff
@@ -15,13 +15,28 @@ export const isAbsoluteUrl = (url: string) => {
   return aux;
 };
 
-export function fixRelativeURLs(htmlString: string, sourceUrl?: string): string {
+export const rebaseRelativeContentLinks = (htmlString: string, sourceUrl?: string) => {
   const domNode = domParser.parseFromString(htmlString, 'text/html');
-  domNode.querySelectorAll('a').forEach((a: HTMLAnchorElement) => {
-    const hrefValue = a.getAttribute('href');
-    if (hrefValue && !isAbsoluteUrl(hrefValue) && sourceUrl) {
-      a.setAttribute('href', fromRelativeUrl(sourceUrl, hrefValue));
+  domNode.querySelectorAll('a').forEach((element: HTMLAnchorElement) => {
+    const hrefValue = element.getAttribute('href');
+    if (hrefValue && !isAbsoluteUrl(hrefValue) &&  sourceUrl) {
+      const resolvedUrl = fromRelativeUrl(sourceUrl, hrefValue);
+      element.setAttribute('href', resolvedUrl);
```

Comment:
there used to be more logic here but now that its been moved you can revert most of these changes